### ISSUE TITLE
[jss-default-unit] values in fallbacks

### ISSUE THREAD

**Issue by ordyakov** [cssinjs/jss#503]
jss-default-unit doesn't process rules in fallbacks due a jss-camel-case plugin.

`root: {
     fallbacks: {
       fontSize: 16,
     },
  }
`

https://github.com/cssinjs/jss-default-unit/blob/master/src/index.js#L32
https://github.com/cssinjs/jss-camel-case/blob/master/src/index.js#L19

**Comment by kof** [comment]
Can you please more specific about what you try to do and what you expected to get?

**Comment by ordyakov** [comment]
I mean that values in fallback rules don't get default units.

This ruleset
```javascript
className: {
  fontSize: 14,

  fallbacks: {
    fontSize: 16,
  },
}
```
transforms to 
```css
.className {
  font-size: 16;
  font-size: 14px;
}
```

**Comment by kof** [comment]
I see, looks like a bug

**Comment by kof** [comment]
fixed in jss-default-unit@6.1.2
